tests/sqlSyncTest.php

```diff
@@ -44,12 +44,13 @@ public function localSqlSync() {
       'yes' => NULL,
     );
     $this->drush('user-create', array($name), $options + array('password' => 'password', 'mail' => $mail));
+    $this->drush('pm-enable', array('comment'), $options + array('yes' => NULL));
```

Comment:
@weitzman The field sanitation is performed only on user fields. So, adding content types will not test the field sanitation. Is there a precedent for adding fields to the user entity in drush testing?